looprpc/client.proto

```diff
@@ -829,6 +829,11 @@ message LiquidityParameters {
     specified by the LoopOutTerms endpoint.
     */
     uint64 max_swap_amount = 15;
+
+    /*
+    The confirmation target for loop in on-chain htlcs.
+    */
+    int32 htlc_conf_target = 17;
```

Comment:
nit: 16?

Reply:
16 is taken by `fee_ppm`, just higher up in the struct for nicer ordering on the cli